proposals/0220-support-for-android-custom-routerservice.md

```diff
@@ -245,35 +242,25 @@ validateAsync method is expected to be called in TransportManager, something lik
 	}
 
 	/**
-	 * start is now synonym of startValidate, and transport.start gets called in startTransport.  
+	 * start does validateAsync first, and then acxtually starts transport after the target RouterService is validated.
```

Comment:
```suggestion
	 * start does validateAsync first, and then actually starts transport after the target RouterService is validated.
```

Reply:
Oops, thanks for pointing that out!
I have updated the source branch to fix the typo.